Snakefile_Regions

```diff
@@ -193,7 +193,7 @@ rule subsample_regions:
         rules.subsample_focus.output.sequences,
         rules.subsample_context.output.sequences
     output:
-        "results/subsampled_alignment{region}.fasta"
+        alignment = "results/subsampled_alignment{region}.fasta"

```

Comment:
If we change this here, do we need to change to `{output.alignment}` in line 201 below?

Reply:
No, since there is only one output, Snakemake will output the single path like this:

```
python3 scripts/combine-and-dedup-fastas.py \
  --input results/subsample_focus_asia.fasta results/subsample_context_asia.fasta \
  --output results/subsampled_alignment_asia.fasta
```

It is good practice to refer to the output by name though, so I should have updated that. I can make a push a little later with that change.

Reply:
Not a bit deal at all! I just wasn't sure and didn't have time to investigate further when I wrote it. :)